app/src/test/java/org/mozilla/fenix/settings/SettingsFragmentTest.kt

```diff
@@ -56,8 +54,6 @@ class SettingsFragmentTest {
         mockkObject(Config)
         every { Config.channel } returns ReleaseChannel.Nightly
 
-        FxNimbus.api = NimbusDisabled(testContext)
-
```

Comment:
What does removing this do exactly?

Reply:
The `api` property is no longer public; this is no longer the way we want to initialize the FxNimbus object.